sourcecode/www/src/components/ResourceModal/ResourceModal.jsx

```diff
@@ -132,16 +132,18 @@ const ResourceModal = ({ isOpen, onClose, resource }) => {
                                 {resource.version.title}
                             </Typography>
                         </Box>
-                        <Box display="flex" marginLeft={1}>
-                            <Typography>
-                                <a
-                                    href={www(`/s/resources/${resource.id}`)}
-                                    target="_blank"
-                                >
-                                    {www(`/s/resources/${resource.id}`)}
-                                </a>
-                            </Typography>
-                        </Box>
+                        {resource.version.isPublished == true &&
```

Comment:
The check is unnecessary, the name of the property infers that it has a boolean value.
When using checks you almost always want to use [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)